src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.EVP.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+using Microsoft.Win32.SafeHandles;
+
+internal static partial class Interop
+{
+    internal static partial class Crypto
+    {
+        [DllImport(Libraries.CryptoNative)]
+        internal extern static SafeEvpMdCtxHandle CreateEvpMdContext();
+
+        [DllImport(Libraries.CryptoNative)]
+        internal extern static int InitializeEvpDigest(SafeEvpMdCtxHandle ctx, IntPtr type, IntPtr impl);
+
+        [DllImport(Libraries.CryptoNative)]
+        internal extern static unsafe int UpdateEvpDigest(SafeEvpMdCtxHandle ctx, byte* d, int cnt);
+
+        [DllImport(Libraries.CryptoNative)]
+        internal extern static unsafe int FinalizeEvpDigest(SafeEvpMdCtxHandle ctx, byte* md, ref uint s);
+
+        [DllImport(Libraries.CryptoNative)]
+        internal extern static void DestroyEvpMdContext(IntPtr ctx);
+
+        [DllImport(Libraries.CryptoNative)]
+        internal extern static int GetEvpMdSize(IntPtr md);
+
+
+        [DllImport(Libraries.CryptoNative)]
+        internal extern static IntPtr CreateEvpMd5();
```

Comment:
Seems happier than "Create" :).  Good with me (it feels a bit lacking, but I'm drawing a blank on a better verb)